src/automation/azext_automation/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isExperimental": true,
-    "azext.minCliCoreVersion": "2.13.0"
+    "azext.minCliCoreVersion": "2.39.0"
```

Comment:
Just out of curiosity, may I ask why are those new features only available to users who have installed the latest version of CLI?

Reply:
This has been auto updated when azdev tools were used. Do you want me to change it back to 2.13.0 ?

Reply:
2.39.0 is required